core/app/models/spree/tax_calculator/default.rb

```diff
@@ -0,0 +1,46 @@
+module Spree
+  module TaxCalculator
+    class Default
+      include Spree::Tax::TaxHelpers
+
+      def initialize(order)
+        @order = order
+      end
+
+      def calculate
+        Spree::Tax::Output::Order.new(
+          id: order.id,
+          line_items: line_item_rates,
```

Comment:
This line reveals that `Spree::Tax::Output::Order#line_items` should be named `line_item_taxes`.